akka-docs/src/main/paradox/java/stream/stages-overview.md

```diff
@@ -4,14 +4,14 @@
 
 ## Source stages
 
-These built-in sources are available from `akka.stream.javadsl.Source`:
+These built-in sources are available from @scala[`akka.stream.scaladsl.Source`] @java[`akka.stream.javadsl.Source`]:
```

Comment:
Actually it will:

![selection_232](https://cloud.githubusercontent.com/assets/131856/26761923/b510a778-4938-11e7-8543-634faa91bf2b.png)
![selection_231](https://cloud.githubusercontent.com/assets/131856/26761926/b972fa3c-4938-11e7-9121-0ecb33101bfe.png)

.. but since it's right after a monospace block it actually looks quite nice :)